README.md

```diff
@@ -47,6 +47,10 @@ Note that the `XXXX-XX-XX` part of the filename is *not* meant to be a placehold
 Let the `status` on `draft` and don't add a `date` field.
 When the PR is merged, GitHub actions will automatically rename the file, add the appropriate date and remove the draft status.
 
+If you are previewing your changes locally, Pelican will not add the new post to the blog unless you set the environment variable `CONTEXT` to `deploy-preview`:
+
+    $ CONTEXT=deploy-preview pelican --listen --autoreload
```

Comment:
```suggestion
    $ CONTEXT=deploy-preview pelican --listen --autoreload
    
Please run the command above *from the root of the repository*, otherwise the build will not succeed.
```